packages/api/test/pin.spec.js

```diff
@@ -0,0 +1,257 @@
+/* eslint-env mocha, browser */
+import assert from 'assert'
+import { endpoint } from './scripts/constants.js'
+import * as JWT from '../src/utils/jwt.js'
+import { SALT } from './scripts/worker-globals.js'
+import { JWT_ISSUER } from '../src/constants.js'
+import { ERROR_CODE, ERROR_STATUS, INVALID_CID, INVALID_META, INVALID_NAME, INVALID_ORIGINS } from '../src/pins.js'
+
+function getTestJWT (sub = 'test', name = 'test') {
+  return JWT.sign({ sub, iss: JWT_ISSUER, iat: 1633957389872, name }, SALT)
+}
+
+describe('POST /pins', () => {
+  let token = null
+  before(async () => {
+    // Create token
+    token = await getTestJWT()
+  })
+
+  it('should receive pin data containing cid', async () => {
+    const res = await fetch(new URL('pins', endpoint).toString(), {
+      method: 'POST',
+      headers: {
+        Authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        cid: 'bafybeibqmrg5e5bwhx2ny4kfcjx2mm3ohh2cd4i54wlygquwx7zbgwqs4e'
+      })
+    })
+
+    assert(res, 'Server responded')
+    assert(res.ok, 'Server response ok')
+    const data = await res.json()
+    assert(data, 'OK')
+  })
+
+  it('requires cid', async () => {
+    const res = await fetch(new URL('pins', endpoint).toString(), {
+      method: 'POST',
+      headers: {
+        Authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({})
+    })
+
+    assert(res, 'Server responded')
+    assert(res.status, `${ERROR_CODE}`)
+    const data = await res.json()
+    const error = data.error
+    assert(error.reason, INVALID_CID)
```

Comment:
Shouldn't this be `REQUIRED_CID`?

`assert()` is an alias for `assert.ok` which checks if the first parameter is truthy. 
Can please review/update the assertions across this file?

Reply:
Good catch... 😭